pyatv/interface.py

```diff
@@ -1249,6 +1249,34 @@ async def text_set(self, text: str) -> None:
         raise exceptions.NotSupportedError()
 
 
+class TouchGestures(ABC):
+    @feature(63, "TouchGesture", "Touch gesture from given coordinates and duration.")
+    async def touch_gesture(self, start_x: int, start_y: int, end_x: int, end_y: int, duration_ms: int) -> None:
+        """ Generate a touch gesture from start to end x,y coordinates (in range [0,1000])
+        in a given time (in milliseconds)
+        :param start_x: Start x coordinate
+        :param start_y: Start y coordinate
+        :param end_x: End x coordinate
+        :param end_y: Endi x coordinate
+        :param duration_ms: Time in milliseconds to reach the end coordinates
+        """
+        raise exceptions.NotSupportedError()
+
+    @feature(64, "TouchEvent", "Touch event to given coordinates.")
+    async def touch_event(self, x: int, y: int, mode: int) -> None:
```

Comment:
Please create an enum for `mode` to maintain proper typing.

Reply:
done